src/containers/AsideNavigation/InlineNavigation.scss

```diff
@@ -0,0 +1,15 @@
+.inline-navigation {
+    padding: 13px 16px 0px;
+
+    &-body {
+        width: fit-content;
```

Comment:
Do we really need this rule?

Reply:
Yep, the component fills all available width otherwise
Might be a nice pr to gravity-ui repo. Or mb I haven't set up the component properly